src/DurableTask.AzureStorage/Tracking/AzureTableTrackingStore.cs

```diff
@@ -980,6 +981,12 @@ public override Task StartAsync()
                         instanceEntity.Properties["Version"] = new EntityProperty(executionStartedEvent.Version);
                         instanceEntity.Properties["CreatedTime"] = new EntityProperty(executionStartedEvent.Timestamp);
                         instanceEntity.Properties["RuntimeStatus"] = new EntityProperty(OrchestrationStatus.Running.ToString());
+                        CorrelationTraceClient.Propagate(() =>
+                        {
+                            historyEntity.Properties["Correlation"] = new EntityProperty(executionStartedEvent.Correlation);
+                            estimatedBytes += System.Text.ASCIIEncoding.ASCII.GetByteCount(executionStartedEvent.Correlation);
```

Comment:
For table storage, I think Unicode is always used.
```suggestion
                            estimatedBytes += Encoding.Unicode.GetByteCount(executionStartedEvent.Correlation);
```